### PR TITLE
UG updated

### PR DESCRIPTION
**Pull request by Cheamybunny**
Removed features table and added app colous and additional details

### CODE REVIEW

File: src/main/java/seedu/modtrek/MainApp.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 import javafx.application.Application;
 import javafx.stage.Stage;
+<<<<<<< HEAD:src/main/java/seedu/modtrek/MainApp.java
```
Comment (jmestxr) [inline src/main/java/seedu/modtrek/MainApp.java:10]: You will have to remove this line. It is the reason the CI checks are failing.

Comment (Cheamybunny) [reply src/main/java/seedu/modtrek/MainApp.java:10]: Ok

**Review by jmestxr — COMMENTED** [review]
Looks good to me at this moment.

**Review by jmestxr — APPROVED** [review]
Looks good to me at this moment

**Pull request merged**